web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -799,17 +799,47 @@ $.extend(fixmystreet.set_up, {
   email_login_form: function() {
     // Log in with email button
     var email_form = $('#js-social-email-hide'),
-        button = $('<button class="btn btn--social btn--social-email">'+translation_strings.login_with_email+'</button>'),
-        form_box = $('<div class="form-box"></div>');
+        button = $('.btn--social-email'),
+        form_box = button.parent();
     button.click(function(e) {
         e.preventDefault();
-        email_form.fadeIn(500);
-        form_box.hide();
+        email_form.show();
+        $('.js-new-report-user-hidden').hide();
+        $('.btn--social').parent().hide();
     });
-    form_box.append(button).insertBefore(email_form);
     if ($('.form-error').length) {
         button.click();
     }
+
+    // Display tweak
+    $('.js-new-report-sign-in-hidden.form-box, .js-new-report-sign-in-shown.form-box').removeClass('form-box');
+
+    $('#js-new-report-user-show').click(function(e) {
```

Comment:
This doesn't fire if you use "Log in with email" if social apps are configured so there's no back button then.